lib/metrics/platforms/linux/stats.js

```diff
@@ -173,13 +173,13 @@ class Stats {
     this.inProgress = false
   }
 
-  start (interval, cb) {
+  start (reportingIntervalInSeconds, cb) {
     if (this.timer) this.stop()
 
     const timer = setInterval(() => {
       if (cb) cb(this.toJSON())
       this.reload()
-    }, interval)
+    }, reportingIntervalInSeconds * 1000)
```

Comment:
Ah, right...forgot about this. I actually fixed it in the runtime metrics PR in a different place. https://github.com/elastic/apm-agent-nodejs/pull/1021/files#diff-a419a7bc99109d34ec9019e4409917fdR27

Reply:
Yeah, I was going back and forth about if I should do it when I called the `createSystemMetrics` function or inside where it used in the `setInterval` function. I settled on the latter because it felt a tiny bit better to me keeping it the same unit until it was specifically needed in another unit.

Reply:
Sounds good. I'll make sure the runtime metrics PR aligns now. :)